### ISSUE TITLE
can't see mAP in tensorboard. Also want to see pr curve

### ISSUE THREAD

**Issue by rkolaf** [tensorflow/models#6410]
#3422 This issue is closed. I want to know how this problem is solved??

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by ruksin** [comment]
Thanks for the reply. What I want is to get the pr curve and map values for each class in tensorboard. While training and when using the tensorboard     `tensorboard --logdir=training_1/`
 I can only see losses, learning rate, queue, batch, target assignment, **I want to see the mAP for each class.** 
Also, **I want to see pr curve to understand the TP, TN, FP, FN, Recall, precision, F1-Score etc.**. But I can only see scalar, graph, distribution, histogram. How can I also add the PR curve under this? I know that we can add it, but I don't know how.

My tensorflow version is 1.5 and I'm using object detection API to do the training using SSD mobilenet models and other models in Tensorflow zoo. I need the above-mentioned values **to compare between the models.**

**Comment by shahryarbaki** [comment]
Where I can see the mAP? On Tensorboard I only have the AP for each class.

**Comment by ebinzacharias** [comment]
How to see the mAP on tensorboard. I can only see the I only see the LearningRate, Losses, batch, global_step and queue in  Tensorboard.

**Comment by shahryarbaki** [comment]
@ebinzacharias Have you specified the `metrics_set` on `eval_config`?

**Comment by ebinzacharias** [comment]
@shahryarbaki  I haven't done that. Thank you for the reply. I will try including metrics_set.

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be an old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
